### ISSUE TITLE
Where do you find the time?

### ISSUE THREAD

**Issue by drpeck** [mattbrailsford/ama#4]
You have a lot of side projects. Do you find you spend all your evenings and weekends on then or are you magic?

**Comment by mattbrailsford** [comment]
Good question, and I actually saw a good [Ted talk](https://www.ted.com/talks/laura_vanderkam_how_to_gain_control_of_your_free_time) on this, but ultimately it comes down to what is a priority to you. If it's important to you, you'll find the time, but I wouldn't feel bad about it if you don't. I think peer pressure can make people feel they should be doing more, but if it's just not your priority, don't worry about it.

It can pretty hard though, and I have come close to burning out several times threatening to quit it all. When I get like that though I tend to take a step back and have a bit of a break, cutting everything back and dropping any commitments I've made and wait for the inspiration to come back again. A big help with the umbraco side of things is having someone to share things with like I do with @LeeKelleher. It really helps knowing you've got someone who's got your back.
